style.css

```diff
@@ -3,18 +3,19 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
 }
 
 body {
   width: 375px;
```

Comment:
- [Optional] Kindly avoid using fixed width in your document. Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels. This will make your page more responsive.